experimental/BasisLieHighestWeight/src/MainAlgorithm.jl

```diff
@@ -156,8 +162,13 @@ function basis_coordinate_ring_kodaira_compute(
         by=(gen -> (sum(coefficients(gen)), reverse(Oscar._vec(coefficients(gen))))),
       )
     end
-
-    V_i = SimpleModuleData(base_lie_algebra(V), i * highest_weight(V))
```

Comment:
Totally off-topic, feel free to ignore me, but I was wondering: Why does this code create the same (???) `V_i` twice? Couldn't you create it before line 135, once?

Reply:
Good catch. This was introduced in https://github.com/oscar-system/Oscar.jl/pull/4886 as part of some refactoring. It shouldn't cost much as this is just a simple wrapper type. But that would certainly be something @janikapeters could clean up